test/readrows.ts

```diff
@@ -317,6 +318,42 @@ describe('Bigtable/ReadRows', () => {
     });
   });
 
+  it('should return row data in the right order', done => {
+    // 1000 rows must be enough to reproduce issues with losing the data and to create backpressure
+    const keyFrom = undefined;
+    const keyTo = undefined;
+    // the server will error after sending this chunk (not row)
+    const errorAfterChunkNo = 100;
+    const dataResults = [];
+
+    service.setService({
+      ReadRows: readRowsImpl2(keyFrom, keyTo, errorAfterChunkNo) as any,
```

Comment:
Can this be changed to something that is not "any" pretty please?

Reply:
Sometimes types can be difficult in tests where we're mocking types, so maybe it can be coerced into a Type or it can be a <Partial>, or at least a comment saying the type!

Reply:
I added a TODO here. When you make this type more specific to actually match the service then there are some more complex compiler errors downstream in the mock server definition to make this work.

Reply:
Just made this work now with a more specific type.